src/loderunner/app/baseline.json

```diff
@@ -2,7 +2,7 @@
   "requests":
   [
     {"path":"/","perfTarget":{"category":"Static"},"validation":{"contentType":"text/html","contains":["<title>Swagger UI</title>","<div id=\"swagger-ui\">"]}},
-    {"path":"/swagger.json","perfTarget":{"category":"Static"},"validation":{"jsonObject":[{"field":"openapi"},{"field":"info"},{"field":"paths"},{"field":"components"}]}},
+    {"path":"/swagger/swagger.json","perfTarget":{"category":"Static"},"validation":{"jsonObject":[{"field":"openapi"},{"field":"info"},{"field":"paths"},{"field":"components"}]}},
```

Comment:
This is the line that I had to change in the docs PR, because swagger is at the root of the current app, but in /wwwroot/swagger of the New App.